src/components/InlineCodeBlock/WrappedText.tsx

```diff
@@ -40,15 +42,46 @@ function containsEmoji(text: string): boolean {
     return CONST.REGEX.EMOJIS.test(text);
 }
 
+/**
+ * Splits long words into multiple strings
+ */
+function splitLongWord(word: string, maxLength: number): string[] {
```

Comment:
Can we add a unit test for this function? With a handful of different sized strings. To make sure it always returns the array we expect.

Reply:
Added unit test for this function 👍